app/models/miq_report_result.rb

```diff
@@ -32,11 +30,16 @@ def result_set
   end
 
   def status
-    MiqTask.human_status(state_or_status)
+    task = MiqTask.find_by(:id=>miq_task_id)
```

Comment:
You should be able to reference the association `miq_task` here instead of doing `MiqTask.find_by(:id=>miq_task_id)`